cpp/ql/lib/change-notes/2024-12-17-template-parameter-base.md

```diff
@@ -0,0 +1,4 @@
+---
+category: feature
+---
+* A new class `TemplateParameterBase` was introduced, which represents C++ non-type template parameters, type template parameters, and template template parameters.
```

Comment:
Technically, this is currently a lie, as nothing represents non-type template parameters. I plan to fix this next.